R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -36,7 +36,7 @@ test_that("Check masked functions", {
                                         any(grepl("=\"ANY\"", capture.output(showMethods(x)[-1])))
                                       }))
   maskedCompletely <- masked[!funcHasAny]
-  namesOfMaskedCompletely <- c("cov", "filter", "sample")
+  namesOfMaskedCompletely <- c("cov", "filter", "sample", "window")
```

Comment:
if this fails that means the current definition will block the stats::window function completely, (ie. user will not be able to call it without altering the code with stats:: prefix)
this is in fact delibrate - to avoid masking the function by accident. for instance, please set the generic (in R/pkg/R/generics.R) as `window(x, ...)` to match the stats package definition - generally that would allow both to be called at the same time.
@sun-rui @shivaram